### DISCUSSION TITLE
尝试加入百度推送代码

### DISCUSSION THREAD

**Discussion by tdouguo** [tangly1024/NotionNext#2086] [Ideas]
**为什么提出这个新的特性改动**
博客的seo 自动推送给百度 对seo 优化好一些

**描述一下你推荐的解决方案**
https://jingyan.baidu.com/article/acf728fd535b99f8e410a377.html
```
import React, { useEffect } from 'react';

function MyComponent() {
  useEffect(() => {
    // 创建脚本标签
    const script = document.createElement('script');
    script.src = '//push.zhanzhang.baidu.com/push.js';
    script.async = true;

    // 插入到文档中
    document.body.appendChild(script);

    // 在组件卸载时移除脚本标签
    return () => {
      document.body.removeChild(script);
    }
  }, []);  // 空的依赖数组表示这个effect只在组件挂载和卸载时运行

  return <div>My Component</div>;
}

export default MyComponent;

```

**描述一下你考虑过的其它替代解决方案**
简要说明你所有想过的有可能解决此问题的方案。

**补充说明**
补充与此特性相关的内容

**Comment by tdouguo** [comment]
https://github.com/tdouguo/NotionNext/tree/add-seo-baidu-push   测试中 稳定了提交pr ，如有需要尝鲜 可合并为的这个分支 （与官方仓库main 只有baidupush 的改动）

**Comment by ghlerrix** [comment]
> https://github.com/tdouguo/NotionNext/tree/add-seo-baidu-push 测试中 稳定了提交pr ，如有需要尝鲜 可合并为的这个分支 （与官方仓库main 只有baidupush 的改动）

这个分支已经删除了啊😂
